### PR TITLE
Add missing values to values.yaml

### PR DESCRIPTION
**Pull request by Dragotic**
This is a suggestion that came up, upon deploying this chart on a sandbox cluster of mine.

As seen in #34 this PR will fix the errors from `helm lint --strict`

### CODE REVIEW

**Comment by desimone** [comment]
Looks reasonable to me, thank you for the PR @Dragotic .

File: values.yaml [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+nameOverride: pomerium
+fullnameOverride: pomerium
+
 # settings that are shared by all services
 config:
   # routes under this wildcard domain are handled by pomerium
   rootDomain: corp.beyondperimeter.com
-  # existingSecret:
-  # existingConfig:
+  existingSecret: ""
```
Comment (desimone) [inline values.yaml]: AFAIK, empty strings are equivalent to [being nil in go templates](https://golang.org/pkg/text/template/#hdr-Actions).

File: values.yaml [hunk 2]
```diff
@@ -1,13 +1,17 @@
 # For detailed explanation of each of the configuration settings see
 # https://www.pomerium.io/reference/
 
+nameOverride: pomerium
```
Comment (desimone) [inline values.yaml]: I'm not sure we want to override the name / full name in it's default configuration. Is this required for passing strict?

File: values.yaml [hunk 3]
```diff
@@ -17,8 +21,8 @@ config:
   policy: {}
 
 authenticate:
-  # fullnameOverride: authenticate
-  # nameOverride: authenticate
+  fullnameOverride: authenticate
```
Comment (desimone) [inline values.yaml]: Same comment as for `name/fullname`

File: values.yaml [hunk 4]
```diff
@@ -125,6 +129,9 @@ extraEnv: {}
 extraArgs: {}
 extraVolumes: {}
 
+annotations: {}
+imagePullSecrets: ""
```
Comment (desimone) [inline values.yaml]: Can you provide a little more context on `imagePullSecrets`? Is this required for lint'ing or?

File: values.yaml (hunk 1 above)
Comment (Dragotic) [reply values.yaml]: Yes, that's correct. Any empty string will result in an empty pipeline value which doesn't generate output.

File: values.yaml (hunk 2 above)
Comment (Dragotic) [reply values.yaml]: Yes, unfortunately, it is required when running `helm lint --strict`.

We can change it to `""` if you think it's better.

File: values.yaml (hunk 3 above)
Comment (Dragotic) [reply values.yaml]: Yes, unfortunately, it is required when running `helm lint --strict`.

We can change it to `""` if you think it's better.

File: values.yaml (hunk 4 above)
Comment (Dragotic) [reply values.yaml]: Same here it is required for lint'ing.

**Comment by Dragotic** [comment]
Yes, I will update the README.

**Comment by desimone** [comment]
> **Update:** I have updated the README. Please note that I didn't add the following to the README, because they are documented on the `values.yaml` and are the default for every helm chart with no difference in the Pomerium Chart:
> 
> ```yaml
> affinity: {}
> tolerations: []
> nodeSelector: {}
> ```

Perfect. I think that makes sense. 

> Also, check that the following descriptions match:
> 
> ```lisp
> config.existingSecret: Name of the existing Kubernetes Secret.
> config.existingConfig: Name of the existing Config Map deployed on Kubernetes.
> config.existingCASecret: Name of the existing CA Secret.
> ```

👍 

LGTM. Just a heads up, might be a few days before this gets merged until I get back home

**Comment by travisgroth** [comment]
@Dragotic Thanks for the PR.  Could you either (a) DCO sign these commits so I can sync this to helm/stable, or (b) if you'd like, mirror your PR there with DCO sign off?

**Comment by Dragotic** [comment]
@travisgroth No problem! I signed-off the two commits. Tell me if something else is missing.

File: values.yaml (hunk 3 above)
Comment (travisgroth) [reply values.yaml]: See other comment on overrides.

File: values.yaml [hunk 5]
```diff
@@ -17,8 +21,8 @@ config:
   policy: {}
 
 authenticate:
-  # fullnameOverride: authenticate
-  # nameOverride: authenticate
+  fullnameOverride: authenticate
+  nameOverride: authenticate
```
Comment (travisgroth) [inline values.yaml]: See other comment on overrides.

File: values.yaml [hunk 6]
```diff
@@ -33,31 +37,31 @@ authenticate:
...
     defaultIPList: []
-  # replicaCount: 1
+  replicaCount: 1
 
 
 authorize:
-  # fullnameOverride: authorize
-  # nameOverride: authorize
+  fullnameOverride: authorize
+  nameOverride: authorize
```
Comment (travisgroth) [inline values.yaml]: See other comment on overrides.

File: values.yaml [hunk 7]
```diff
@@ -33,31 +37,31 @@ authenticate:
...
     defaultIPList: []
-  # replicaCount: 1
+  replicaCount: 1
 
 
 proxy:
-  # fullnameOverride: proxy
-  # nameOverride: proxy
+  fullnameOverride: proxy
+  nameOverride: proxy
```
Comment (travisgroth) [inline values.yaml]: See other comment on overrides.

File: values.yaml (hunk 2 above)
Comment (victornoel) [reply values.yaml]: yes I think so too, I made the same comment on the helm repository actually :)

**Comment by Dragotic** [comment]
@victornoel @travisgroth Thanks for the comments guys. Check again for the new commits.

**Comment by travisgroth** [comment]
Merged in https://github.com/helm/charts/pull/19527.  👍

**Pull request merged**